compiler-core/src/language_server/reference.rs

```diff
@@ -267,6 +271,32 @@ pub fn reference_for_ast_node(
             location: *name_location,
             target_kind: RenameTarget::Definition,
         }),
+        Located::ModuleName { location, name, .. } => Some(Referenced::ModuleName {
+            name: name.clone(),
+            location,
+        }),
+        Located::ModuleStatement(Definition::Import(
+            import @ Import {
+                location,
+                module,
+                as_name,
+                ..
+            },
+        )) => match as_name {
+            Some(alias) => Some(Referenced::ModuleName {
+                name: module.clone(),
+                // alias location includes the "as " part
+                location: SrcSpan::new(alias.1.start + 3, alias.1.end),
```

Comment:
There could be any number of spaces around the `as` so we cannot safely add 3 to it here

Reply:
Either seems good to me!